tests/framework/extensions/clusters/clusters.go

```diff
@@ -138,6 +140,63 @@ func CheckServiceAccountTokenSecret(client *rancher.Client, clusterName string)
 	}
 }
 
+// CreateRancherBaselinePSACT creates custom PSACT called rancher-baseline which sets each PSS to baseline.
+func CreateRancherBaselinePSACT(client *rancher.Client, psact string) error {
+	_, err := client.Steve.SteveType(PodSecurityAdmissionSteveResoureType).ByID(psact)
+	if err == nil {
+		return nil
+	}
```

Comment:
```suggestion
	if err != nil {
		return err
	}
```

Reply:
Is this correct or a typo?

Reply:
Corrected it

Reply:
Once you're done with PR comments. We'll need to do another freeform.